docs/groovy-script/mods/selectionguicrafting/recipe.md

```diff
@@ -0,0 +1,255 @@
+---
+title: "Crafting Recipe"
+titleTemplate: "Selection GUI Crafting | CleanroomMC"
+description: "Creates a recipe that is shown in the specified category. Each recipe requires at least an input (placed in the offhand), tool (placed in the mainhand), and output. There can also be an optional catalyst (placed in the inventory). The recipe can have its own frame, sounds, particles, and progress bar. It also allows to specify how the sounds are played, if the recipe can be added to the crafting queue, how the output items are handed to the player, and how much durability is consumed, if the tool is a damageable item. The crafting time as well as a possible XP reward can be set as well."
+source_code_link: "https://github.com/Ender-Development/selection-gui-crafting-continued/blob/master/src/main/java/io/enderdev/selectionguicrafting/integration/groovyscript/Recipe.java"
+---
+
+# Crafting Recipe (Selection GUI Crafting)
+
+## Description
+
+Creates a recipe that is shown in the specified category. Each recipe requires at least an input (placed in the offhand), tool (placed in the mainhand), and output. There can also be an optional catalyst (placed in the inventory). The recipe can have its own frame, sounds, particles, and progress bar. It also allows to specify how the sounds are played, if the recipe can be added to the crafting queue, how the output items are handed to the player, and how much durability is consumed, if the tool is a damageable item. The crafting time as well as a possible XP reward can be set as well.
```

Comment:
i think this `placed in the inventory` -> `placed anywhere in the inventory` would make the requirement clearer.
same thing as in `category`:
```suggestion
Creates a recipe that is shown in the specified category. Each recipe requires at least an input (placed in the offhand), tool (placed in the mainhand), and output. There can also be an optional catalyst (placed in the inventory). The recipe can have its own frame, sounds, particles, progress bar, can specify how the sounds are played, if the recipe can be added to the crafting queue, how the output items are handed to the player, how much durability is consumed if the tool is a damageable item, the crafting time, and how much XP is rewarded.
```
should also note that many of these have a fallback based on the category.